.github/workflows/update-feeds.yml

```diff
@@ -0,0 +1,364 @@
+            --update-limit 200
+          echo "::endgroup::"
+        continue-on-error: true
+
+      - name: Cleanup private key
+        if: always()
+        run: rm -f $CLOUDKIT_PRIVATE_KEY_PATH
+
+  # Stale feeds (daily)
+  update-stale:
+    needs: [determine-tier, build]
+    if: needs.determine-tier.outputs.runs_stale == 'true'
+    runs-on: ubuntu-latest
+    timeout-minutes: 120
+
+    steps:
+      - name: Download binary artifact
+        uses: actions/download-artifact@v4
+        with:
+          name: celestra-cloud-binary
+          path: ./bin
+
+      - name: Make binary executable
+        run: chmod +x ./bin/celestra-cloud
+
+      - name: Create CloudKit private key file
+        run: |
+          echo "${{ secrets.CLOUDKIT_PRIVATE_KEY }}" > $CLOUDKIT_PRIVATE_KEY_PATH
+          chmod 600 $CLOUDKIT_PRIVATE_KEY_PATH
+
+      - name: Update stale feeds
+        run: |
+          echo "::group::Stale Feeds Update"
+          # Calculate date 7 days ago in ISO8601 format (Linux date command)
+          WEEK_AGO=$(date -u -d '7 days ago' '+%Y-%m-%dT%H:%M:%SZ')
+          echo "Updating feeds not attempted since: $WEEK_AGO"
+          ./bin/celestra-cloud update \
+            --update-last-attempted-before "$WEEK_AGO" \
+            --update-max-failures 10 \
+            --update-delay 3.0 \
+            --update-limit 300
+          echo "::endgroup::"
+        continue-on-error: true
+
+      - name: Cleanup private key
+        if: always()
+        run: rm -f $CLOUDKIT_PRIVATE_KEY_PATH
+
+  # PR integration test (limited scope)
+  update-pr-test:
+    needs: [determine-tier, build]
+    if: needs.determine-tier.outputs.runs_pr_test == 'true'
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+
+    steps:
+      - name: Download binary artifact
+        uses: actions/download-artifact@v4
+        with:
+          name: celestra-cloud-binary
+          path: ./bin
+
+      - name: Make binary executable
+        run: chmod +x ./bin/celestra-cloud
+
+      - name: Create CloudKit private key file
+        run: |
+          echo "${{ secrets.CLOUDKIT_PRIVATE_KEY }}" > $CLOUDKIT_PRIVATE_KEY_PATH
+          chmod 600 $CLOUDKIT_PRIVATE_KEY_PATH
+
+      - name: Run PR integration test
+        run: |
+          echo "::group::PR Integration Test"
+          echo "Environment: $CLOUDKIT_ENVIRONMENT (development)"
+          echo "Testing with limited feeds (smoke test)"
+          echo ""
+          ./bin/celestra-cloud update \
+            --update-limit 5 \
+            --update-max-failures 0 \
+            --update-delay 1.0
+          echo "::endgroup::"
+        continue-on-error: false
+
+      - name: Cleanup private key
+        if: always()
+        run: rm -f $CLOUDKIT_PRIVATE_KEY_PATH
+
+  # Summary report
+  summary:
+    needs: [determine-tier, build, update-high-priority, update-standard, update-stale, update-pr-test]
+    if: always()
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Generate summary
```

Comment:
I'd like either in a separate file or here more details such as detailed markdown file with feeds updated, errors, successes, etc...